packages/esm-appointments-app/translations/es.json

```diff
@@ -136,7 +135,7 @@
   "unscheduled": "No programada",
   "unscheduledAppointments": "Citas no programadas",
   "unscheduledAppointments_lower": "citas no programadas",
-  "upcoming": "Próximas",
+  "upcomming": "Upcoming",
```

Comment:
I don't know where this typo is coming from, but we should fix it.  It's a typo _and_ causing the translation to get lost.

Reply:
Ugh... TBH all these changes were auto-created by the commit hooks, and I just committed them assuming they were correct.  Bad call on my part... I'll revert all these changes before merging and we can look into all this separetly.

Reply:
I had the same experience, with this exact same message code when I committed recently.  I thought I fixed it in the code, but I guess not.  Best to figure out where this misspelling is originating from and fix it.  It's someone else's bug that is affecting your commit at this point.

Reply:
Yeah, I remember you mentioned it but guess I didn't take it to heart... will clean up it, will take a small amount of work on my part because my local branch is currently a little out  sync (perhaps due to another issue)